common/flatpak-dir.c

```diff
@@ -3383,7 +3383,9 @@ repo_pull_one_local_untrusted (FlatpakDir          *self,
         }
     }
 
+#ifdef FLATPAK_ENABLE_P2P
   collection_id = flatpak_dir_get_remote_collection_id (self, remote_name);
```

Comment:
While reviewing this I noticed that `flatpak_dir_get_remote_collection_id()` doesn’t squash `""` to `NULL` like `repo_get_remote_collection_id()` does. Might make sense to do a follow-up commit which does that.

Reply:
Good point. I pushed a fixup to use `repo_get_remote_collection_id` here and I'll make another PR for that issue.